reference/vane-apis/iris.md

```diff
@@ -7,7 +7,18 @@ template = "doc.html"
 # Iris
 
 In this document we describe the public interface for Iris. Namely, we describe
-each `task` that Iris can be `%pass`ed, and which `gift`(s) Iris can `%give` in return.
+each `task` that Iris can be `%pass`ed, and which `gift`(s) Iris can `%give` in
+return.
+
+## Returns to Unix
+
+Much of what Iris does is translating `task`s from other vanes into `card`s
+meant for Unix via the `%request` and `%cancel-request` `task`s. What this means in practice is that Iris may be `%pass`ed a
+`task` via some `duct`, which in response Iris then `%give`s a `gift` to Unix
+via another `duct`. This `gift` will actually be a response to the `%born`
+`task` that Iris was `%pass`ed when the Urbit was resumed, which came along the
+duct to Unix. This is not an ideal situation but also one not easily remedied.
```

Comment:
I'd lose the last sentence.  I think it's basically righteous for vanes to be translators between urbit-space and unix.

Reply:
OK I removed it. I think I must still be misunderstanding something here though. I didn't think having one task trigger a give response to another task was proper form. Is this actually the norm and I've built up the wrong idea in my head?

Reply:
Sometimes it's the right thing, yeah.  Subscriptions commonly do this for example: A subscribes to B on duct /foo, then C does something to B on duct /bar that updates the subscription, so B gives an update on /foo to A (and maybe some kind of acknowledgment on /bar to C).

Ducts are conceptual stack traces not bound by mechanical constraints like the literal nock stack trace you see when something crashes.  So conceptually, when the runtime sends a `%born` event, it's saying "I'm running an HTTP client, so lmk if you want to send any requests".  This "causes" Iris to use that whenever someone asks it to perform an HTTP request.